accepted/2021/statics-in-interfaces/README.md

```diff
@@ -2,13 +2,12 @@
 
 **DRAFT**
 
-**Libraries PM** [Immo Landwerth](https://github.com/terrajobst)
-**Language PM** [Mads Torgersen](https://github.com/MadsTorgersen)
+**Owners** [Immo Landwerth](https://github.com/terrajobst), [Mads Torgersen](https://github.com/MadsTorgersen)
```

Comment:
Right now, the regex looks for pipe separated authors. This happens to work because it matches a single owner that has the value `[Immo Landwerth](https://github.com/terrajobst), [Mads Torgersen](https://github.com/MadsTorgersen)`, but that's probably not what we want. I suggest to either update the regex to include support for both pipes and commas or just change the document to use the pipe.

Reply:
Good catch; thanks.